### ISSUE TITLE
two cable detected

### ISSUE THREAD

**Issue by youthy** [Maxx53/NoCableLauncher#47]
![image](https://user-images.githubusercontent.com/4058883/30069609-c9d10cda-9293-11e7-953d-22c5e7c8e33d.png)
I use maya22 , I only use one cable, but why detect two?  the guitar rig 5 works fine.

**Comment by PvlMksv** [comment]
+1 UP
I have the same one. My soundcard is Creative SB X-Fi. I think we have this bullshit 'cause we have more than one recording device on the same PID/VID. I've switched off everything but not the Creative X-Fi microphone with no desired result. 
NoCableLauncher date is 28/12/2016
Life's pain :(

**Comment by PvlMksv** [comment]
Manual PIDs from Creative X-Fi- no result. "Realtone Cable is unplugged blah blah blah..."

**Comment by PvlMksv** [comment]
Microphone (Creative X-Fi) is MMDEVAPI\AudioEndpoints
so it's unreachable in manual pid mode :(
Life's pain

**Comment by Maxx53** [comment]
Yes, if your physical device has 2 or more inputs, only default input is used by rocksmith

**Comment by PvlMksv** [comment]
Ok. When I switched off all the rec. Devices except microphone. Mic became default input, but i still have two cables. I need help what have i lurking for 😊

**Comment by Maxx53** [comment]
Is It your device?
http://blog-imgs-52.fc2.com/p/u/r/puredigital/SR51P-3.jpg

What about other effect-modeling soft, e.g. Guitar Rig? Can you play through your card?
In what channel do you hear your guitar?
![image](https://user-images.githubusercontent.com/5123763/30269918-5bf3349a-96f3-11e7-90a4-762dc2d7afb2.png)

**Comment by PvlMksv** [comment]
Ok. I'll check tonight.

**Comment by youthy** [comment]
![image](https://user-images.githubusercontent.com/4058883/30312827-c88a241c-97cd-11e7-932f-45cde7228944.png)
I use this device.
I will see the Guitar Rig channel this nignt when I go home.

**Comment by Maxx53** [comment]
@youthy most likely, a microphone input (**left** channel) will be used by rocksmith.

**Comment by youthy** [comment]
@Maxx53 But I don't have a microphone=.=.  I only use the HEADPHONE and the Hi-z for guitar input

**Comment by Maxx53** [comment]
@youthy I use "XLR to Jack" cable to connect guitar in microphone input. There's no other way on my soundcard.
![image](https://user-images.githubusercontent.com/5123763/30320767-50fe6bee-97bc-11e7-8926-fd11924df2f4.png)

**Comment by youthy** [comment]
@Maxx53 hello.my Guitar rig both L, R can work, even, I turn off them.I still can make sound....I don't know why.How can I use my maya22 in Rocksmith QAQ

**Comment by youthy** [comment]
@Maxx53 SORRY! I am wrong! my guitar rig doesn't make sound. ... because I close it I can still make guitar work. It's my sound card that make sound==
